packages/hooks/src/hooks/useContract.ts

```diff
@@ -2,6 +2,16 @@ import React from 'react';
 import { Web3Context } from '../Provider';
 import { Contract } from 'ethers';
 
+/**
+ * @dev Hook to get the contract instance from ABI and contract address
```

Comment:
```suggestion
 * Gets an instance of a contract from its ABI and address.
```

`jsdoc` doesn't have an `@dev` directive. The lines of text without a directive are used as the description.

![image](https://user-images.githubusercontent.com/1954752/148595534-446ae2f9-1386-41b3-bcbd-233a3863766c.png)